metadata/src/main/java/org/apache/kafka/controller/ReplicationControlManager.java

```diff
@@ -1364,7 +1364,25 @@ Boolean isBrokerUnfenced(int brokerId) {
                 setErrorCode(apiError.error().code()).
                 setErrorMessage(apiError.message()));
         }
-        return new ControllerResult<>(records, results, true);
+        StringBuilder resultsBuilder = new StringBuilder();
```

Comment:
nit: would it be enough to print out the `results` directly. I think `CreatePartitionsTopicResult` already gives us a reasonable `toString`.